action.yml

```
@@ -0,0 +1,17 @@
+name: 'Determine Next Version Mobile App'
+description: 'Parses Git Tags that start with v and determines the next version'
+author: 'Gopal Sharma <gps@surya-soft.com>'
+inputs:
+  GH_TOKEN:
+    description: GitHub Token used to interact with GitHub API
+    required: true
+  TAG_PREFIX:
+    description: Prefix to look for in version tags
+    required: true
+    default: v
+outputs:
+  NEXT_BUILD_VERSION:
+    description: Next build number to use
+runs:
+  using: node12
+  main: index.js
```

@gps  Since PR is from forked repository the CI is not triggered and distribution is not generated. 
After distribution is generated this should be `'dist/index.js'`
```suggestion
  main: 'dist/index.js'
```